### PR TITLE
test: add coverage

### PR DESCRIPTION
**Pull request by GuillaumeCisco**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Create automatic tests
- [x] No automatic tests failures
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -33,3 +33,4 @@ gem 'base32', '0.3.2'
 gem 'rotp', '3.1'
 gem 'httparty', '0.13.7'
 gem 'ipaddress', '0.8.3'
+gem 'simplecov', require: false, group: :test
```
Comment (arnaudbesnier) [inline Gemfile:36]: You should put this gem definition in "test" group at line 19, to ease the readability.

File: Gemfile.lock [hunk 2]
```diff
@@ -135,20 +137,25 @@ GEM
...
       sprockets (>= 3.0.0)
     sqlite3 (1.3.13)
-    thor (0.20.0)
+    thor (1.0.1)
     thread_safe (0.3.6)
-    tzinfo (1.2.5)
+    tzinfo (1.2.8)
       thread_safe (~> 0.1)
-    useragent (0.16.5)
+    useragent (0.16.10)
```
Comment (arnaudbesnier) [inline Gemfile.lock:158]: Why all those updates?
This is a quite hazardous operation (especially when you upgrade to new major versions)
And I don't see the relation with the coverage impllementation.

**Review by arnaudbesnier — CHANGES_REQUESTED** [review]

**Comment by arnaudbesnier** [comment]
Is this coverage run by the CI?

File: Gemfile.lock (hunk 2 above)
Comment (GuillaumeCisco) [reply Gemfile.lock:158]: Looks like `bundle install` auto update these dependencies

**Comment by GuillaumeCisco** [comment]
> Is this coverage run by the CI?

no, it is still a work in progress to show what could be done

**Comment by GuillaumeCisco** [comment]
It should be ok now.
I cleaned and added [codecov](https://app.codecov.io/gh/ForestAdmin/forest-rails/branch/chore-coverage) for travis.
Maybe we also need to bind code climate to this repository :)

File: README.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Forest Admin in Rails
 [![Gem](https://badge.fury.io/rb/forest_liana.svg)](https://badge.fury.io/rb/forest_liana)
-[![CI status](https://travis-ci.org/ForestAdmin/forest-rails.svg?branch=devel)](https://travis-ci.org/ForestAdmin/forest-rails)
+[![CI status](https://travis-ci.org/ForestAdmin/forest-rails.svg?branch=master)](https://travis-ci.org/ForestAdmin/forest-rails)
+[![codecov](https://codecov.io/gh/ForestAdmin/forest-rails/branch/master/graph/badge.svg)](https://codecov.io/gh/ForestAdmin/forest-rails)
```
Comment (arnaudbesnier) [inline README.md:4]: Why do we use both Codecov and Code Climate? We should clean the Codecov badge.

Comment (GuillaumeCisco) [reply README.md:4]: We should remove it :)

**Review by arnaudbesnier — APPROVED** [review]
Looks great! 🎖️ 
Let's wait the last commit to be CI-ed to see if the coverage is still reported.

**Pull request merged**